static/js/publisher/release/components/releasesConfirm.js

```diff
@@ -56,207 +53,95 @@ class ReleasesConfirm extends Component {
-                        {release.channel} to{" "}
-                        <b>
-                          {release.progressive.changes
-                            .map(change => {
-                              if (change.key === "paused") {
-                                return change.value === true
-                                  ? "paused"
-                                  : "resumed";
-                              }
-                              return `${change.value}%`;
-                            })
-                            .join(" at ")}
-                        </b>
-                        <br />
-                      </span>
-                    );
-                  })}
-                  <br />
-                </span>
-              </span>{" "}
-              to update.
-            </Fragment>
-          )}{" "}
-          {cancelProgressiveCount > 0 && (
-            <Fragment>
-              <span className="p-tooltip">
-                <span className="p-help">
-                  {cancelProgressiveCount} release
-                  {cancelProgressiveCount > 1 ? "s" : ""}
-                </span>
-                <span className="p-tooltip__message" role="tooltip">
-                  {Object.keys(cancelProgressive).map(revId => {
-                    const release = cancelProgressive[revId];
-
-                    return (
-                      <span key={revId}>
-                        <b>{release.revision.revision}</b> (
-                        {release.revision.version}) on{" "}
-                        {release.revision.architectures.join(", ")}{" "}
-                        {release.channel}
-                        <br />
-                      </span>
-                    );
-                  })}
-                </span>
-              </span>{" "}
-              to cancel.
-            </Fragment>
-          )}
-        </div>
-        <div className="col-6 p-releasses-confirm__actions">
-          {showProgressive && (
-            <ProgressiveConfirm
-              percentage={this.state.percentage}
-              newReleases={newReleasesToProgress}
-              onChange={this.onPercentageChange.bind(this)}
-            />
-          )}
-          <div className="p-releases-confirm__buttons">
-            <button
-              className="p-button--neutral u-no-margin--bottom"
-              disabled={!isCancelEnabled}
-              onClick={this.onRevertClick.bind(this)}
-            >
-              Revert
-            </button>
-            <button
-              className="p-button--positive is-inline u-no-margin--bottom u-no-margin--right"
-              disabled={!isApplyEnabled}
-              onClick={this.onApplyClick.bind(this)}
-            >
-              {isLoading ? "Loading..." : "Save"}
-            </button>
+      <Fragment>
+        <div className="p-releases-confirm">
+          {showDetails && <ReleasesConfirmDetails updates={updates} />}
```

Comment:
Won't comment on the design to much, but it felt a bit unexpected that the details appear on top of confirm.
They also feel a bit detached.